ffi/capi/dart/package/lib/src/lib.g.dart

```diff
@@ -0,0 +1,12137 @@
+// https://github.com/dart-lang/sdk/issues/53946
```

Comment:
We often add a comment to generated files, describing how and where they are generated, so that this can be reproduced.